diff --git a/test.c b/test.c
index 4f8d516..084d96e 100644
--- a/test.c
+++ b/test.c
@@ -7,5 +7,28 @@ int main()
 	printf("HI i am chandan");
 	printf("HI i am chandan");
 	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
+	printf("HI i am chandan");
 	return 0;
 }
